livre/eq/Channel.cpp

```diff
@@ -118,8 +147,9 @@ struct Channel::Impl
 public:
     explicit Impl( Channel* channel )
           : _channel( channel )
-          , _frameInfo( Frustum( Matrix4f(), Matrix4f()), INVALID_FRAME )
+          , _frameInfo( Frustum( Matrix4f(), Matrix4f()), INVALID_FRAME, INVALID_FRAME )
           , _progress( "Loading bricks", 0 )
+          , _frameCounter( INVALID_FRAME )
```

Comment:
After looking more into the code now I have a better suggestion which is getting rid of FrameInfo. Let's discuss it, but I'm pretty sure you can do it.